scripts/onload.js

```diff
@@ -256,8 +257,8 @@ map.on('load', function () {
 		map,
 		{
 			'sourceName': 'school_house_senate_districts_UNION',
-			'sourceID': 'school_house_senate_districts_UNION',
-			'sourceURL': 'mapbox://core-gis.a81c8ecf',
+			'sourceID': 'school_house_senate_districts-dpf66q',
```

Comment:
It turns out that the source ID within the tileset had changed.  Luckily I could just download a single tile and look at it in QGIS to find the new one.

Reply:
Awesome, thank you and well done!